### PR TITLE
Fix syntax - minor improvements

### PR DESCRIPTION
**Pull request by GeoDoo**

### CODE REVIEW

**Comment by francesco-strazzullo** [comment]
@GeoDoo can you please squash the commit in this PR? We are trying to keep a clean history

File: README.md [hunk 1]
```diff
@@ -2,42 +2,41 @@
...
+The Frameworkless Movement is a group of developers interested in developing applications without frameworks. We don't hate frameworks, nor we will ever create campaigns against frameworks, but we perceive the **abuse of frameworks as a lack of knowledge regarding technical debt** and the availability of alternatives given by the vanilla language or by dedicated libraries.
 
 ## Why the Movement
 
-Everytime that a team is using a framework, it is also taking a **risk**. This risk is, after some time, to be locked up with a tool that is not giving the team any kind of value and that most of the time it represent a major roadblock to change. Most importantly a framework could "die" way before the software that uses it, leaving the developers with a heavy burden.
+Every time a team uses a framework, it also takes a **risk**. The risk is that after some time has passed the team ends up with a tool that does not provide any kind of value anymore and that most of the times represents a major roadblock to change. Most importantly a framework could "die" way before the software that uses it, leaving the developers with a heavy burden.
```
Comment (francesco-strazzullo) [inline README.md:9]: @GeoDoo why did you remove `locked`? I think it gives quite the idea...it also resemble the [Vendor Lock-in](https://en.wikipedia.org/wiki/Vendor_lock-in) anti-pattern.

File: README.md [hunk 2]
```diff
@@ -2,42 +2,41 @@
...
+Frameworkless **does not mean "reinvent the wheel"**. There are a lot of "tools" (frameworks, libraries but also StackOverflow) that solve known and common problems in a great way. Frameworkless means, however, not to assume that in the software I'm writing I can afford to delegate this responsibility and depend on other software (a library for example) to solve a specific problem. Or worse: don't reinvent the wheel and bind hand and foot to a whole framework even if I don't really need it.
 
-Frameworkless **does not mean that you must develop all software by yourself without a framework** in every context and in every situation, thinking that this is more secure and more performant ([Not Invented Here syndrome](https://en.wikipedia.org/wiki/Not_invented_here#In_computing)), far from it.
+Frameworkless **does not mean that you must develop all software by yourself without a framework** in every context and in every situation, thinking that this is more secure and more high performance ([Not Invented Here syndrome](https://en.wikipedia.org/wiki/Not_invented_here#In_computing)), far from it.
```
Comment (francesco-strazzullo) [inline README.md]: "this is more secure and more high performance" Could be changed in "more secure and with higher perfomances"?

**Comment by francesco-strazzullo** [comment]
@GeoDoo thanks for the squash. I just left some notes on your comment.

File: README.md (hunk 1 above)
Comment (GeoDoo) [reply README.md:9]: @francesco-strazzullo If you write it like this, you will see that somehow the meaning is that you get locked inside somewhere with a tool... It is not the best syntax in my opinion to convey the message that you create a bottleneck or roadblock or that you get stuck in a situation and you cannot escape it easily! If you check the meaning of lock up you will understand maybe better what I am saying

http://www.wordreference.com/enit/lock%20up

Comment (GeoDoo) [reply README.md:9]: but of course we can probably put the pattern in parentheses or sth if you want

Comment (GeoDoo) [reply README.md:9]: and notice the lock *in* for the pattern

damned english phrasal verbs

File: README.md (hunk 2 above)
Comment (GeoDoo) [reply README.md]: high perfomance is an adjective... I am not 100% how you write it, maybe with a - between the two words.

But if we go with your advice it should be in singular: performance i think

File: README.md (hunk 1 above)
Comment (GeoDoo) [reply README.md:9]: So, in the end i changed the meaning a lil bit meaning that you are eventually left to deal with a deprecated tool! I do not know if you want to keep it or change it back to lock up (maybe we can change lock up => lock in) I dunno for sure

:)

Comment (francesco-strazzullo) [reply README.md:9]: @lorenzomassacci @adellava @Violo George convinced me here. What do you think?

File: README.md (hunk 2 above)
Comment (francesco-strazzullo) [reply README.md]: It seems that you need a dash here. Can you update it? https://www.macmillandictionary.com/dictionary/british/high-performance

Comment (GeoDoo) [reply README.md]: @francesco-strazzullo Done!

File: README.md (hunk 1 above)
Comment (adellava) [reply README.md:9]: ok for me! thx @GeoDoo

Comment (Violo) [reply README.md:9]: Ok for me too! Great @GeoDoo
